### DISCUSSION TITLE
How often the nuget package is updated?

### DISCUSSION THREAD

**Discussion by markand-bhatt** [mkaring/ConfuserEx#473] [Q&A] [answered]
How often the nuget package is updated?
I see on nuget.org, Confuser.MSBuild package was last updated on 18/Apr/2019

**Comment by mkaring** [comment, marked as answer]
There was a very old and very dead prerelease version listed at the top of the list. I removed that. The current one is now the 1.6.0.
The packages are updated with every release as part of the release  process.
